src/fluree/db/query/fql/syntax.cljc

```diff
@@ -44,8 +44,8 @@
    [:map-of ::json-ld-keyword :any]
    [:fn
     {:error/fn
-     (fn [{:keys [value]} _]
-       (str "Query: " (pr-str value) " does not have exactly one select clause. "
+     (fn [_ _]
+       (str "Query does not have exactly one select clause. "
```

Comment:
Does the offending query still get shown to the user elsewhere?

Reply:
In the "Provided: ..." output, perhaps?

Reply:
Yep! I removed it from this fn because it was getting duplicated by the `"Provided"` part of the formatted message

Reply:
Those errors look like this now

```clj
  (syntax/coerce-query '{:select [?s]
                         :selectOne [?s ?p ]
                         :where  [[?s ?p ?o ]]})

   ;; =>
   ;; Query does not have exactly one select clause. One of 'select', 'selectOne',
   ;; 'select-one', 'selectDistinct', or 'select-distinct' is required in queries.
   ;; See documentation here for more details:
   ;; https://next.developers.flur.ee/docs/reference/errorcodes#query-missing-select;
   ;; Provided: {:select [?s], :selectOne [?s ?p], :where [[?s ?p ?o]]}
```